wicamo96/FieldScout
# Greenhouse management

# Description

Added functionality to Greenhouse Management page which allows you to create new houses associated with your facility, edit house name, and delete unwanted houses.

## Type of change


# Testing Instructions

### Create database and populate data if necessary
- If you haven't created the FieldScout database, please do so in visual studio community.
- Use this to fill out tables and columns: [SQL data](https://github.com/wicamo96/FieldScout/blob/main/SQL/01_FieldScout_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/wicamo96/FieldScout/blob/main/SQL/02_FieldScout_Seed_Data.sql)

### Testing Greenhouse Management CRUD
- Click the execute with debugger solid green button to launch the backend in visual studio community.
- From the client directory in a git terminal, run `npm run dev`
- Once the Vite + React window pops up, you should be directed to the log in page
- Log in with the default email
- Navigate to Greenhouse Management via the link in the navbar
- You should see a list containing all the houses associated with your facility. (Default is House A, B, and C)
- Click the Add New House button at the bottom of the table
- Enter a name in the modal and click Add
- Your new house should show in the list
- Click the Edit button next to your test house, change the name in the modal, and select Submit Edit
- Your edit should immediately reflect in the list
- Click the Delete button next to your test house and select Confirm Delete
- Your deletion should immediately reflect in the list

# Checklist:
